cfme/common/provider_views.py

```diff
@@ -166,10 +166,33 @@ def is_displayed(self):
                 self.navigation.currently_selected == ['Compute', 'Clouds', 'Providers'])
 
 
+class PhysicalProviderDetailsToolBar(ProviderDetailsToolBar):
+    """
+    Represents the physical provider toolbar and its controls
+    """
+    reload = Button(title='Reload Current Display')
```

Comment:
This is already defined in ProviderDetailsToolBar, where its version picked, and should be accessible without any change to your calls to this widget, and without re-defining the widget in this class.  Looks like you can use ProviderDetailsToolBar for PhysicalProviderDetailsView.

https://github.com/ManageIQ/integration_tests/pull/5968/files#diff-2a1de88a15dbb2a71c0fe4de72721d6dR94

Reply:
@mshriver I had to redefine this here. For some reason in my UI, the title is 'Reload Current Display'; however, it is defined as 'Reload current display' in ProviderDetailsToolBar, so my test case was failing because it could not find the reload button because of the case difference. What is the correct way to handle this? Thanks

Reply:
Noted - what appliance version are you running? I believe that case change was an older one?

Otherwise - if its different in upstream/58z/59z, we need to update the `VersionPick`  dictionary in ProviderDetailsToolBar to include a new key/value pair for that appliance version.

Reply:
@mshriver How do I tell what appliance version I am running? I ran _appliance_console_ and under _Summary Information_ it says _master_ for the version. I believe I installed this appliance a month ago, so it probably needs updated.

Reply:
@mshriver I think that the name of the appliance I deployed was "manageiq-vsphere-master-20171113-f9f5aef4e1.ova", so maybe it was one of the nightly builds. I'll try running it against a newer appliance. Thanks